crates/hc_sandbox/tests/cli.rs

```diff
@@ -1271,14 +1271,8 @@ async fn get_config_root_path(child: &mut Child) -> PathBuf {
 
     while let Ok(Some(line)) = lines.next_line().await {
         println!("@@@-{line}-@@@");
-        if line.contains("Created [ConfigRootPath(\"") {
-            let start = line
-                .find("(\"")
-                .expect("Unexpected config root path line format.");
-            let end = line
-                .find("\")]")
-                .expect("Unexpected config root path line format.");
-            return PathBuf::from(&line[(start + 2)..end]);
+        if line.contains("0:") {
```

Comment:
Probably this should be

```suggestion
        if line.starts_with("0:") {
```

Because you're assuming it starts at 2.